### PR TITLE
(Added)[PXN-2979] - Reenviar campos del checkout al endpoint de remedies

### PR DESCRIPTION
**Pull request by GiselaARamosC**
## What have changed?

- En la response de checkout se agregaron los campos payment_method_name y bank_info, que luego seran utilizados para armar el request al endpoint de remedies

### CODE REVIEW

File: MercadoPagoSDK/MercadoPagoSDK/Services/Models/PXCustomOptionSearchItem.swift [hunk 1]
```diff
@@ -15,8 +15,10 @@ open class PXCustomOptionSearchItem: NSObject, Codable {
     open var lastFourDigits: String
     open var escStatus: String?
     open var displayInfo: PXPayerPaymentMethodDisplayInfo?
+    open var paymentMethodName: String?
+    open var bankInfo: PXPayerPaymentMethodBankInfo?
 
-    public init(id: String, description: String?, paymentMethodId: String?, paymentTypeId: String?, discountInfo: String?, defaultAmountConfiguration: String?, amountConfigurations: [String: PXAmountConfiguration]?, comment: String?, issuer: PXIssuer?, firstSixDigits: String, lastFourDigits: String, escStatus: String?, displayInfo: PXPayerPaymentMethodDisplayInfo?) {
+    public init(id: String, description: String?, paymentMethodId: String?, paymentTypeId: String?, discountInfo: String?, defaultAmountConfiguration: String?, amountConfigurations: [String: PXAmountConfiguration]?, comment: String?, issuer: PXIssuer?, firstSixDigits: String, lastFourDigits: String, escStatus: String?, displayInfo: PXPayerPaymentMethodDisplayInfo?, paymentMethodName: String?, bankInfo: PXPayerPaymentMethodBankInfo?) {
```
Comment (rgrajalesmeli) [inline MercadoPagoSDK/MercadoPagoSDK/Services/Models/PXCustomOptionSearchItem.swift:21]: A very minor comment, can we split this line so we can avoid warnings?

File: MercadoPagoSDK/MercadoPagoSDK/Services/Models/PXCustomOptionSearchItem.swift [hunk 2]
```diff
@@ -67,8 +73,10 @@ open class PXCustomOptionSearchItem: NSObject, Codable {
...
+        let paymentMethodName: String? = try container.decodeIfPresent(String.self, forKey: .paymentMethodName)
+        let bankInfo: PXPayerPaymentMethodBankInfo? = try container.decodeIfPresent(PXPayerPaymentMethodBankInfo.self, forKey: .bankInfo)
 
-        self.init(id: id, description: description, paymentMethodId: paymentMethodId, paymentTypeId: paymentTypeId, discountInfo: discountInfo, defaultAmountConfiguration: defaultAmountConfiguration, amountConfigurations: amountConfigurations, comment: comment, issuer: issuer, firstSixDigits: firstSixDigits, lastFourDigits: lastFourDigits, escStatus: escStatus, displayInfo: displayInfo)
+        self.init(id: id, description: description, paymentMethodId: paymentMethodId, paymentTypeId: paymentTypeId, discountInfo: discountInfo, defaultAmountConfiguration: defaultAmountConfiguration, amountConfigurations: amountConfigurations, comment: comment, issuer: issuer, firstSixDigits: firstSixDigits, lastFourDigits: lastFourDigits, escStatus: escStatus, displayInfo: displayInfo, paymentMethodName: paymentMethodName, bankInfo: bankInfo)
```
Comment (rgrajalesmeli) [inline MercadoPagoSDK/MercadoPagoSDK/Services/Models/PXCustomOptionSearchItem.swift:79]: same here

**Review by rgrajalesmeli — APPROVED** [review]
LGTM! just a very minor comment. It can be ignore if no needed.

File: MercadoPagoSDK/MercadoPagoSDK/Services/Models/PXCustomOptionSearchItem.swift (hunk 1 above)
Comment (GiselaARamosC) [reply MercadoPagoSDK/MercadoPagoSDK/Services/Models/PXCustomOptionSearchItem.swift]: done

File: MercadoPagoSDK/MercadoPagoSDK/Services/Models/PXCustomOptionSearchItem.swift (hunk 2 above)
Comment (GiselaARamosC) [reply MercadoPagoSDK/MercadoPagoSDK/Services/Models/PXCustomOptionSearchItem.swift]: done

**Review by rgrajalesmeli — APPROVED** [review]
LG!!

**Pull request merged**